src/routes/(authed)/layout.server.test.ts

```diff
@@ -0,0 +1,43 @@
+import { describe, it, expect, vi } from 'vitest';
+import { redirect } from '@sveltejs/kit';
+import { load } from './+layout.server';
+
+vi.mock('@sveltejs/kit', () => ({
+  redirect: vi.fn().mockImplementation(() => {
+    const error = new Error('Redirect');
+    // eslint-disable-next-line
+    (error as any).status = 303;
+    // eslint-disable-next-line
+    (error as any).location = '/auth';
+    throw error;
+  }),
+}));
+
+describe('load', () => {
+  it('should redirect to /auth if user is not present', async () => {
+    const mockEvent = { locals: { user: null } };
+
+    try {
+      await load(mockEvent);
+    } catch (error) {
+      console.log('Error here');
```

Comment:
It is unnecessary, but it's better not to bloat code with `console.log`s

Also, recently, I found out this about construction `await expect(() => throwFn()).rejects.to...`, but your code is solid as well